packages/block-library/src/post-terms/edit.js

```diff
@@ -36,6 +36,7 @@ export default function PostTermsEdit( {
 			if ( ! term ) return {};
 			const taxonomies = select( coreStore ).getTaxonomies( {
 				per_page: -1,
+				context: 'view',
```

Comment:
I think using "context" in the data module causes issues where callers of. `getTaxonomy` in other components might receive an incomplete taxonomy while they expected a result similar to "context: edit"

Having better support for "context" in the data module is something I wish we'd introduce, the solution might be to consider the "context" argument similarly to "_fields", meaning the result should be considered incomplete.

Alternatively, we can use apiFetch directly like we already in the PostTerms related components.

Reply:
For now I used `apiFetch` here: https://github.com/WordPress/gutenberg/pull/32947/commits/d444b21462865a5abe85add63fdbc66166db7b9d